share/getters/utils.go

```diff
@@ -84,9 +84,10 @@ func collectSharesByNamespace(
 		return nil, err
 	}
 
-	// return ErrNamespaceNotFound if no shares are found for the namespace.ID
+	// return ErrNamespaceNotFound along with collected proofs if no shares are found for the
+	// namespace.ID
```

Comment:
We should add this special case to the documentation of the interface.

Reply:
The error is defined right next to interface and has same description.

Reply:
// ErrNamespaceNotFound is returned by GetSharesByNamespace when data for requested root does
// not include any shares from the given namespace

From main

I don't see the comment being changed in this PR

Reply:
Also, mentioning the special case just on the error is not enough

Reply:
I've updated the description of ErrNamespaceNotFound to indicate how it is supposed to be handled along with non-inclusion proofs by user.